libs/core/dtos/anime-params.dto.ts

```diff
@@ -0,0 +1,9 @@
+/** Anime params. */
+export type AnimeParamsDto = Readonly<{
```

Comment:
```suggestion
export type AnimePaginationDto = Readonly<{
```
This is too abstract a name for the structure.